app/bundles/EmailBundle/Helper/MailHelper.php

```diff
@@ -913,7 +917,16 @@ public function addTo($address, $name = null)
         $this->checkBatchMaxRecipients();
 
         try {
-            $this->message->addTo((new AddressDTO($address, $name))->toMailerAddress());
+            $fullAddress          = (new AddressDTO($address, $name))->toMailerAddress();
+            $encodedAddressLength = strlen((new MailboxListHeader('To', [$fullAddress]))->getBodyAsString());
```

Comment:
Shouldn't  be here mb_strlen?

Reply:
That's a good question, I've been thinking about it and in this case, `strlen()` should be the correct choice because:
1. We're measuring the length of the encoded string, not the original human-readable string.
2. The encoded string is always ASCII, even if it represents non-ASCII characters.
3. Email servers and standards deal with these encoded lengths, not the decoded character counts.